packages/zotero.rb

```diff
@@ -0,0 +1,127 @@
+      system "git config submodule.xpi/zotero-transfw.url \"#{$srcdir}/zotero-transfw\""
+      system "git -c protocol.file.allow=always submodule update"
+      system "echo 'Step 1: PASSED'"
+    }
+    
+    Dir.chdir("zotero") {
+      system "git submodule init"
+      system "git config submodule.translators.url '#{$srcdir}/translators'"
+      system "git config submodule.styles.url '#{$srcdir}/bundled-styles'"
+      system "git config submodule.pdf-worker.url '#{$srcdir}/pdf-worker'"
+      system "git config submodule.note-editor.url '#{$srcdir}/note-editor'"
+      system "git config submodule.pdf-reader.url '#{$srcdir}/pdf-reader'"
+      system "git config submodule.resource/schema/global.url '#{$srcdir}/zotero-schema'"
+      system "git config submodule.resource/SingleFile.url '#{$srcdir}/SingleFile'"
+      system "git config submodule.chrome/content/zotero/xpcom/utilities.url '#{$srcdir}/utilities'"
+      system "git config submodule.chrome/content/zotero/xpcom/translate.url '#{$srcdir}/translate'"
+      system "git config submodule.chrome/content/zotero/locale/csl.url '#{$srcdir}/locales'"
+      system "git -c protocol.file.allow=always submodule update"
+      system "echo 'Step 2: PASSED'"
+
+      Dir.chdir("pdf-reader") {
+        system "git submodule init"
+        system "git config submodule.pdf.js.url '#{$srcdir}/pdf.js'"
+        system "git -c protocol.file.allow=always submodule update"
+      }
+
+      Dir.chdir("pdf-worker") {
+        system "git submodule init"
+        system "git config submodule.pdf.js.url '#{$srcdir}/pdf.js'"
+      }
+
+      system "echo 'Installing LTS node.js...'"
+
+      system "nodebrew install v18.16.1 || true" # if node version is already present or locale is not correct somehow it'd throw an error and exit the script
+      system "nodebrew exec v18.16.1 npm i --legacy-peer-deps"
+    }
+
+    system "ls -l -a"
+
+    Dir.chdir("zotero-standalone-build") {
+      system "git submodule init"
+      system "git config submodule.modules/zotero-libreoffice-integration.url '#{$srcdir}/zotero-libreoffice-integration'"
+      system "git -c protocol.file.allow=always submodule update"
+      system "echo Fetching for standalone building..."
+      system "./fetch_xulrunner.sh -p l"
+      system "./fetch_pdftools"
+    }
+
+    system "mv zotero zotero-client"
+
+    Dir.chdir("zotero-client") {
+      system "Actually building..."
+      system "NODE_OPTIONS=--openssl-legacy-provider nodebrew exec v18.16.1 npm run build"
+    }
+
+    Dir.chdir("zotero-standalone-build") {
+      system "scripts/dir_build -p l"
+    }
+  end
+
+  def self.install                 # the steps required to install the package
+    system "install -dDm755 #{CREW_DEST_PREFIX}/{bin,lib/zotero}"
+    system "mv '#{$srcdir}/zotero-standalone-build/staging/Zotero_linux-x86_64'/* '#{CREW_DEST_PREFIX}/lib/zotero'"
+    system "ln -s '#{CREW_DEST_PREFIX}/lib/zotero/zotero' '#{CREW_DEST_PREFIX}/bin/zotero'"
```

Comment:
The source of the symlink needs to be in the regular `CREW_LIB_PREFIX` not the `CREW_DEST` location - otherwise the symlink breaks upon installation:
```
    FileUtils.ln_s "#{CREW_LIB_PREFIX}/zotero/zotero", "#{CREW_DEST_PREFIX}/bin/zotero"
```